pkg/frontend/mysql_cmd_executor.go

```diff
@@ -191,6 +193,96 @@ func (o *outputQueue) flush() error {
 	return nil
 }
 
+/*
+handle show create table in plan2 and tae
+*/
+func handleShowCreateTable2(obj interface{}, bat *batch.Batch) (*batch.Batch, error) {
+	ses := obj.(*Session)
+	var vecs []*vector.Vector
+	col := bat.Vecs[0].Col.(*types.Bytes)
+	dbName := string(col.Data[col.Offsets[0] : col.Offsets[0]+col.Lengths[0]])
+
+	col = bat.Vecs[1].Col.(*types.Bytes)
+	tableName := col.Data[col.Offsets[0] : col.Offsets[0]+col.Lengths[0]]
+
+	//get database
+	storage := ses.GetStorage()
+	txnHandler := ses.GetTxnHandler()
+	db, err := storage.Database(dbName, txnHandler.GetTxn().GetCtx())
+	if err != nil {
+		return nil, err
+	}
+
+	//get table
+	table, err := db.Relation(string(tableName), txnHandler.GetTxn().GetCtx())
```

Comment:
You do not need read the attributes again. The rewritten sql has been executed and done for you.